src/app/Http/ViewHelpers/Employee/EmployeeLogViewHelper.php

```diff
@@ -49,4 +50,20 @@ public static function employee(Employee $employee): array
             'avatar' => ImageHelper::getAvatar($employee, 80),
         ];
     }
+
+    /**
+     * Log Types for the employee.
+     */
+    public static function employeeLogTypes(Employee $employee): array
+    {
+        $logTypes = DB::table('employee_logs')
+            ->select('action')
+            ->selectRaw('count(*) as number_of_logs')
+            ->where('author_id', $employee->id)
```

Comment:
Same here, it looks like you're only counting logs for the employee that they "authored" themselves

Reply:
This again was my understanding of what logs should be returned. I've updated this to count logs based on the employee_id instead of author_id.